.github/workflows/create-stable-release.yml

```diff
@@ -9,7 +9,7 @@ jobs:
   create-release:
     permissions:
       contents: write
-    uses: workleap/wl-reusable-workflows/.github/workflows/create-stable-release.yml
+    uses: workleap/wl-reusable-workflows/.github/workflows/create-stable-release.yml@main
```

Comment:
Just an idea: Maybe we should have created it using the `action.yml` _format_ to simplify usage a little bit, similar to https://github.com/workleap/wl-reusable-workflows/blob/main/send-slack-notification/action.yml